test/e2e/resource_quota.go

```diff
@@ -41,6 +41,75 @@ const (
 
 var classGold string = "gold"
 
+var _ = framework.KubeDescribe("ResourceQuota [Feature:LocalStorageCapacityIsolation]", func() {
```

Comment:
why is this new code a separate `KubeDescribe`? I'm not sure it matters much, but rather than a new KubeDescribe I think you could have added `Context()`s under the existing KubeDesribe.

Reply:
yeah, same reason to limitrange, i want to merge them together when the feature graduates to beta or GA